esx_service/utils/auth_data_test.py

```diff
@@ -476,19 +485,19 @@ def test_list_tenants(self):
                                    default_privileges[auth_data_const.COL_MAX_VOLUME_SIZE],
                                    default_privileges[auth_data_const.COL_USAGE_QUOTA]
                                    ]
-        tenant2_actual_output = [tenants_list[1].default_privileges[0][auth_data_const.COL_TENANT_ID],
-                                 tenants_list[1].default_privileges[0][auth_data_const.COL_DATASTORE],
-                                 tenants_list[1].default_privileges[0][auth_data_const.COL_CREATE_VOLUME],
-                                 tenants_list[1].default_privileges[0][auth_data_const.COL_DELETE_VOLUME],
-                                 tenants_list[1].default_privileges[0][auth_data_const.COL_MOUNT_VOLUME],
-                                 tenants_list[1].default_privileges[0][auth_data_const.COL_MAX_VOLUME_SIZE],
-                                 tenants_list[1].default_privileges[0][auth_data_const.COL_USAGE_QUOTA]
+        tenant2_actual_output = [tenants_list[tenant2_idx].default_privileges[0][auth_data_const.COL_TENANT_ID],
+                                 tenants_list[tenant2_idx].default_privileges[0][auth_data_const.COL_DATASTORE],
+                                 tenants_list[tenant2_idx].default_privileges[0][auth_data_const.COL_CREATE_VOLUME],
+                                 tenants_list[tenant2_idx].default_privileges[0][auth_data_const.COL_DELETE_VOLUME],
+                                 tenants_list[tenant2_idx].default_privileges[0][auth_data_const.COL_MOUNT_VOLUME],
+                                 tenants_list[tenant2_idx].default_privileges[0][auth_data_const.COL_MAX_VOLUME_SIZE],
+                                 tenants_list[tenant2_idx].default_privileges[0][auth_data_const.COL_USAGE_QUOTA]
```

Comment:
"test_vmdkops_on_fallback_tenant_vm" did this.
1. It test the create, attach, detach, remove command when FALLBACK tenant and FALLBACK privileges are present
2. It then test the vmdk ops after removing FALLBACK tenant and FALLBACK privilege

What do you mean "you need to check for fallback tenant and privileges being created in the DB as a separate test"? 
FALLBACK tenant and privileges are created when table is created.